subsys/bluetooth/shell/bt.c

```diff
@@ -1777,6 +1777,34 @@ static int cmd_past_unsubscribe(const struct shell *shell, size_t argc,
 
 	return err;
 }
+
+static int cmd_per_adv_sync_transfer(const struct shell *shell, size_t argc,
+				     char *argv[])
+{
+	int err;
+	int index = 0;
+	struct bt_le_per_adv_sync *per_adv_sync = NULL;
```

Comment:
Unnecessary initialisation upon declaration since this gets unconditionally assigned to anyway in the function